src/components/SponsorsBanner.astro

```diff
@@ -141,7 +141,7 @@ function formatCurrency(value: number): string {
   const goal = 12000;
```

Comment:
This goal should now come from JSON, right?

Reply:
Yes, the goal should come from json. Thanks for catching that.